src/com/googlecode/jmxtrans/model/output/OpenTSDBWriter.java

```diff
@@ -113,6 +113,11 @@ protected void startOutput() throws IOException {
 	@Override
 	protected void sendOutput(String metricLine) throws IOException {
 		try {
+			
+			if (isDebugEnabled() || log.isDebugEnabled()) {
+				log.debug("Sending to server {}:{} line={}", host, port,metricLine);
```

Comment:
I agreed with you.  It is finally fixed !  Do you have any suggestion on how eclipse template that auto correct this kind of stuff ?  Just asking so cmd + f will auto fix this.

Thanks,
Bhavesh